foxpuppet/windows/browser/navbar.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from foxpuppet.region import Region
+
+
+class NavBar(Region):
+
+    """Representation of the NavBar
+    :param selenium: WebDriver Object
+    :param root: Root WebElement
+    :type selenium:
+        :py:class:`~selenium.webdriver.remote.webdriver.WebDriver` object
+    :type root:
+        :py:class:`~selenium.webdriver.remote.webdriver.WebDriver` object
+    """
+
+    _root_locator = (By.ID, 'nav-bar')
+    _tracking_protection_shield_locator = (By.ID, 'tracking-protection-icon')
+
+    def __init__(self, selenium, *args, **kwargs):
+        self.selenium = selenium
+
+    @property
+    def is_tracking_shield_displayed(self):
+        """Tracking Protection shield
+        :returns:
+            :py:class:`~selenium.webdriver.remote.webdriver.WebDriver` object
+        :return type: object
+        """
+        with self.selenium.context(self.selenium.CONTEXT_CHROME):
+            if self.selenium.find_element(
+                *self._tracking_protection_shield_locator). \
+                    get_attribute('state'):
+                return True
+            return False
```

Comment:
> Sure, but you really want a method for each needed property/attribute? Wow!

No, this is being used as an indication that the icon is displayed. If it wasn't for the bug related to `is_displayed` for chrome elements, we'd likely be using that.